pkg/diag/validator/config_connector.go

```diff
@@ -153,15 +154,16 @@ func (s *configConnectorResourceStatus) updateAE(errCode proto.StatusCode, msg s
 type configConnectorSelector struct {
 	client    kubernetes.Interface
 	dynClient dynamic.Interface
-	uResource unstructured.Unstructured
+	resources schema.GroupVersionKind
```

Comment:
How about the following renames:
- rename this `resources` field &rarr; `kind`, since it isn't a set of resources
- rename this `configConnectorSelector` type to `resourceSelector` (and related objects), since there's nothing inherent about CC